src/app/ConfirmCallDestination/_components/ActionButton.tsx

```diff
@@ -0,0 +1,46 @@
+'use client';
+
+import { Button, Flex } from '@kuma-ui/core';
+import { useRouter } from 'next/navigation';
+
+const ActionButtons = () => {
+  const router = useRouter();
+
+  const handleCancel= () => {
+    router.back();
+  };
+
+  const handleConfirm = () => {
+  };
+
+  return (
+    <Flex justifyContent="space-around">
+      <Button
+        backgroundColor="#E4007F"
+        border={'none'}
```

Comment:
propsに文字列を渡す際に`""`だったり`{''}`だったりしているのは意図しているところですか？
`{}`で囲われていると変数なのかな？と錯覚するので、できたら`""`に統一したいです。

eslintを設定するのが良いかな？と思います。
https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-curly-brace-presence.md

```suggestion
        border="none"
```